README.md

```diff
@@ -203,20 +208,27 @@ Note that Valkey Over RDMA is an experimental feature.
 It may be changed or removed in any minor or major version.
 Currently, it is only supported on Linux.
 
-To manually run a Valkey server with RDMA mode:
+* RDMA built-in mode:
+    ```
+    ./src/valkey-server --protected-mode no \
+         --rdma-bind 192.168.122.100 --rdma-port 6379
+    ```
 
-    % ./src/valkey-server --protected-mode no \
-         --loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379
+* RDMA module mode:
+    ```
+    ./src/valkey-server --protected-mode no \
+         --loadmodule src/valkey-rdma.so --rdma-bind 192.168.122.100 --rdma-port 6379
+    ```
 
 It's possible to change bind address/port of RDMA by runtime command:
 
-    192.168.122.100:6379> CONFIG SET rdma.port 6380
```

Comment:
`rdma.port` was the module-defined config, which is removed now because there are normal configs instead.

Before this PR, the RDMA module created module configs in OnLoad. Now, even if RDMA is built as a module, it is using the real configs instead of module-defined config.

The removed module configs is a breaking change, but RDMA is experimental, so it's no problem.

Reply:
so, now if we are using module mode, the old rdma.port is not valid anymore? Yes, it is a breaking change and we should mention it in the release notes (thought it is experimental)

Reply:
> so, now if we are using module mode, the old rdma.port is not valid anymore? Yes, it is a breaking change and we should mention it in the release notes (thought it is experimental)

Right. `rdma.port` is totally removed.

Reply:
> Yes, it is a breaking change and we should mention it in the release notes (thought it is experimental)

It doesn't look good that we have a section of breaking changes in the minor version. We have to write it in a smart way so that it doesn't look bad.

Reply:
I would just not list it as a breaking change. I would just call it out as a change in the RDMA code, which is still experimental.

Reply:
I guess alternatively, can we make rdma-port an alias for rdma.port?

Reply:
We have already use `--rdma-bind 192.168.122.100 --rdma-port 6379` instead of `--loadmodule src/valkey-rdma.so port=6379 bind=192.168.122.1` since this change, should we (and other few users in use) really care this dynamic config...

Reply:
Yeah, that's true. I'm okay with this as is, I just thought I would bring it up.

Reply:
If RDMA would not be considered experimental, then we would have had to care about this. It's great that we called it experimental. :smile:

Reply:
> I would just not list it as a breaking change. I would just call it out as a change in the RDMA code, which is still experimental.

I removed the breaking-change label again.